content/en/blog/_posts/2023-04-11-kmsv2-beta.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: blog
+title: "Kubernetes 1.27: KMS V2 Moves to Beta"
+date: 2023-04-11
+slug: kms-v2-moves-to-beta
+---
+
+**Authors:** Anish Ramasekar, Mo Khan, and Rita Zhang (Microsoft)
+
+With Kubernetes 1.27, we are moving Key Management Service (KMS) v2 API to beta.
```

Comment:
Is “we” Microsoft, or Kubernetes? Consider making it clear.

Reply:
Updated it to make it clear. I used the similar pattern that was used in few other blogs. LMK if this is ok!